cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,16 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
+
+    cy.visit('https://conduit.mate.academy/user/register');
+
+    cy.get(':nth-child(1) > .form-control').type(user.username);
```

Comment:
Using CSS selectors directly can make your tests fragile and hard to maintain. If the structure of your HTML changes, your tests might break. Consider using `data-cy` or `data-test` attributes to select elements. For example, you could add `data-cy="username-input"` to your input field in your HTML, and then select it in your test with `cy.get('[data-cy=username-input]')`.